training/src/pages/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './TextFieldDemo';
```

Comment:
export {default as TextFieldDemo } from './TextFieldDemo'; 
Because when you will be adding multiple folders to this pages folder then it would be more clear 
do this in the component folder too